content/200-orm/200-prisma-client/600-observability-and-logging/250-opentelemetry-tracing.mdx

```diff
@@ -145,6 +154,63 @@ registerInstrumentations({
 provider.register()
 ```
 
+This approach provides maximum flexibility but may involve additional configuration steps.
+
+#### Option 2: Using `@opentelemetry/sdk-node`
+
+For many users, especially beginners, the `NodeSDK` class simplifies OpenTelemetry setup by bundling common defaults into a single, unified configuration.
+
+```ts
+// Imports
+import { OTLPTraceExporter } from '@opentelemetry/exporter-trace-otlp-proto'
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { PrismaInstrumentation } from '@prisma/instrumentation'
+
+// Configure the OTLP trace exporter
+const traceExporter = new OTLPTraceExporter({
+  url: 'https://api.honeycomb.io/v1/traces', // Replace with your collector's endpoint
+  headers: {
+    'x-honeycomb-team': 'HONEYCOMB_API_KEY', // Replace with your Honeycomb API key or collector auth header
+  },
+})
+
+// Initialize the NodeSDK
+const sdk = new NodeSDK({
+  serviceName: 'my-service-name', // Replace with your service name
+  traceExporter,
+  instrumentations: [
+    new PrismaInstrumentation({
+      middleware: true, // Enable middleware tracing if needed
+    }),
+  ],
+})
+
+// Start the SDK
+sdk.start()
+
+// Handle graceful shutdown
+process.on('SIGTERM', async () => {
+  try {
+    await sdk.shutdown()
+    console.log('Tracing shut down successfully')
+  } catch (err) {
+    console.error('Error shutting down tracing', err)
+  } finally {
+    process.exit(0)
+  }
+})
+```
+
+Choose the `NodeSDK` approach if:
+* You are starting with OpenTelemetry and want a simplified setup.
+* You need to quickly integrate tracing with minimal boilerplate.
+* You are using an OTLP-compatible tracing backend like Honeycomb, Jaeger, or Datadog.
+
+Choose the `NodeTracerProvider` approach if:
+* You need detailed control over how spans are created, processed, and exported.
+* You are using custom span processors or exporters.
+* Your application requires specific instrumentation or sampling strategies.
```

Comment:
for consistency with the rest of the content, I suggest using hyphens for bullets instead of asterisks:

```suggestion
Choose the `NodeSDK` approach if:
- You are starting with OpenTelemetry and want a simplified setup.
- You need to quickly integrate tracing with minimal boilerplate.
- You are using an OTLP-compatible tracing backend like Honeycomb, Jaeger, or Datadog.

Choose the `NodeTracerProvider` approach if:
- You need detailed control over how spans are created, processed, and exported.
- You are using custom span processors or exporters.
- Your application requires specific instrumentation or sampling strategies.
```